src/annotations/standard/PropertyAnnotation.php

```diff
@@ -14,12 +14,81 @@
 namespace mindplay\annotations\standard;
 
 use mindplay\annotations\Annotation;
+use mindplay\annotations\IAnnotationParser;
+use mindplay\annotations\AnnotationException;
+use mindplay\annotations\IAnnotationFileAware;
+use mindplay\annotations\AnnotationFile;
 
 /**
  * Defines a magic/virtual property and it's type
- *
- * @todo implement this
+ * @usage('class'=>true, 'inherited'=>true)
  */
-class PropertyAnnotation extends Annotation
+class PropertyAnnotation extends Annotation implements IAnnotationParser, IAnnotationFileAware
 {
+	/**
+	 * @var string Specifies the property type
+	 */
+	public $type;
+	/**
```

Comment:
There should be an empty line between property/method names.

P.S.
Also change in similar places.